### PR TITLE
Feature/pack list view

### PR DESCRIPTION
**Pull request by rilifon**
Now we have a screen showing all your packs instead of just opening a random pack.

Added a pack image and collections now can have an icon to display in its pack.

TODO in future PRs, add an open-pack animation and color option to collections

### CODE REVIEW

File: game/database/schema/collection.lua [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 return {
   { id = 'name', name = "Full Name", type = "string" },
+  { id = 'image', name = "Image", type = 'enum', options = 'resources.texture' },
```
Comment (Kazuo256) [inline game/database/schema/collection.lua]: You don't use this field, right?

Comment (rilifon) [reply game/database/schema/collection.lua]: I use it to draw the pack icon

Comment (Kazuo256) [reply game/database/schema/collection.lua]: Oh, right, Nevermind.

Comment (rilifon) [reply game/database/schema/collection.lua]: Line 189 of file packlist.lua

Comment (rilifon) [reply game/database/schema/collection.lua]: I'll rebase the branch as soon as possible ^^

Comment (Kazuo256) [reply game/database/schema/collection.lua]: Yes, I meant to say it was me who got it wrong. My bad.

Comment (rilifon) [reply game/database/schema/collection.lua]: No prob ^^

**Pull request merged**